shared/util/arrays.js

```diff
@@ -16,3 +16,7 @@ export function intersperseFn (separatorFn: (index: number, x: any) => any, arr:
   }, [arr[0]])
 }
 
+export function filterNull<X> (arr: Array<?X>): Array<X> {
+  // $FlowIssue can't understand this
+  return arr.filter(i => !!i)
+}
```

Comment:
Lodash has a function for this called [_.compact](https://lodash.com/docs#compact).
> Creates an array with all falsey values removed. The values false, null, 0, "", undefined, and NaN are falsey.